### DISCUSSION TITLE
[Transactions] Manage friends

### DISCUSSION THREAD

**Discussion by SirSimon04** [SE-TINF22B2/G1-CashCompass#32] [User stories]
### Description
As a user, I want to add friends so that I can add them to a shared transaction. I also want to delete friends if we are not friends anymore :(

**Story points**: 5
### Acceptance criteria
- [ ] add friends with a code each user has
- [ ] view all friends in a list
- [ ] delete friends
- [ ] display all transactions with a friend

**Comment by SirSimon04** [comment]
Prio has been changed to low, this is now a would.

**Comment by stefanrudi** [comment]
Changed the category of the user story from Friends to Transactions, so it fits more to the big picture.
